assets/styles/app.css

```
@@ -0,0 +1,16 @@
+@import "tailwindcss";
+@tailwind utilities;
```

Stylelint (17.4.0)

[error] Unexpected unknown at-rule "@tailwind" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)